spec/factories/users.rb

```diff
@@ -4,6 +4,16 @@
     sequence(:uid)    { |i| "12345#{i}" }
     sequence(:name)   { |i| "name#{i}" }
     sequence(:email)  { |i| "email#{i}@example.test" }
-    provider { 'github' }
+    provider  { 'github' }
+    token     { 'valid_token' }
+  end
```

Comment:
```suggestion
  trait(:with_invalid_token) do
    token 'invalid token'
  end
  end
```

Reply:
It simplifies your trait structure a lot: instead of two copies you just override the selected attributes in a nested trait, and then do `create(:user, :with_invalid_token)` when you need to invoke the trait.